docs/content/deployment/guides/kubernetes/deploying-with-helm.mdx

```diff
@@ -264,6 +264,38 @@ dagster-user-deployments:
 
 `dagsterApiGrpcArgs` also supports loading repository definitions from a package name. To find the applicable arguments, [read here](/concepts/repositories-workspaces/workspaces#running-your-own-grpc-server).
 
+You can also specify configmaps, secrets, volumes, and labels for individual user code deployments, and use the `includeConfigInLaunchedRuns` key to tell Dagster to include that same configuration on any runs launched from that user code deployment:
```

Comment:
wording feels a little weird since the runs are launched from the daemon?

```suggestion
You can also specify configmaps, secrets, volumes, and labels for individual user code deployments, and use the `includeConfigInLaunchedRuns` key to tell Dagster to include that same configuration on any runs launched from jobs in that user code deployment:
```